server/monitor/src/main/java/org/apache/accumulo/monitor/util/Table.java

```diff
@@ -227,4 +247,25 @@ private static void row(StringBuilder sb, boolean highlight, ArrayList<TableColu
     sb.append("</tr>\n");
   }
 
+  private static void doDropdownMenu(String redir, String page, String tableName, StringBuilder sb, SortedMap<String,String> namespaces, String namespace) {
+
+    String namespaceUrl = String.format("/op?action=namespace&redir=%s&page=%s&table=%s&selected=", redir, page, tableName);
+
+    sb.append("<ul id=\"namespaces\">\n");
+    sb.append("<li><a ").append(namespace.equals("*") ? "class=\"active\" " : "").append("href=\"").append(namespaceUrl).append("*\">*</a></li>");
+    for (String key : namespaces.keySet()) {
+      if (key.equals("")) {
+        sb.append("<li><a ").append(namespace.equals("-") ? "class=\"active\" " : "").append("href=\"").append(namespaceUrl).append("-\">-</a></li>");
```

Comment:
Same as above. `-` is a little confusing.  Could add text describing this option as default tables.